apisix/plugins/limit-count/util.lua

```diff
@@ -0,0 +1,85 @@
+--
```

Comment:
`apisix/plugins/limit-count/util.lua`

I checked `_M.incoming` and `_M.log_phase_incoming`, and they both work for `redis`.

We should add `redis` to the filename of the function name.